components/camel-cdi/src/main/java/org/apache/camel/cdi/CdiEventConsumer.java

```diff
@@ -37,8 +37,8 @@
     }
 
     @Override
-    protected void doStart() throws Exception {
-        super.doStart();
+    protected void doInit() throws Exception {
+        super.doInit();
```

Comment:
This should be in doStart as we add / remove the consumer when the consumer is started/stopped, eg you can stop a consumer/route and then later start it. So this should be as before

Reply:
Ok, I get it and I have not noticed that this is an event